apps/rehabstod/src/pages/CurrentSickLeaves/CurrentSickLeaves.tsx

```diff
@@ -58,48 +59,57 @@ export function CurrentSickLeaves() {
         />
       </div>
       <PrintFilters isDoctor={isDoctor} />
+      {error ? (
+        <DisplayError
+          heading="Sjukfall för enheten kunde inte hämtas."
+          errorType="error"
+          text="Sjukfall för enheten kan inte visas på grund av ett tekniskt fel. Försök igen om en stund. Om felet kvarstår, kontakta i första hand din lokala IT-support och i andra hand"
+          dynamicLink
+        />
+      ) : (
```

Comment:
This becomes a huge inline if-statement. Perhaps it could be better too simply.

```tsx
{error && <DisplayError ....>}
{!error && <Table ...>}
```